docs/_extra/api-reference/schemas/group-schema.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
If I understand correctly this `group-schema.json` is in here but it's not currently used yet? (`profile-group-schema.json` is used instead, for now)

Reply:
Exactly! I'm able to keep the future group shape in `group-schema` but I can't do the same with the future parameters as they live in `hypothesis.yaml`—so you'll just have to trust me that `document_uri` will show up.